lib/logstash/environment.rb

```diff
@@ -114,6 +95,42 @@ def bundler_setup!(options = {})
       ::Bundler.setup
     end
 
+    def runtime_jars_root(dir_name, package)
```

Comment:
Ok, lets do it that way so we can move on and merge, we are always allowed to revert the change if need.